xbmc/addons/InputStream.cpp

```diff
@@ -36,8 +36,14 @@ std::unique_ptr<CInputStream> CInputStream::FromExtension(AddonProps props, cons
   std::string extensions = CAddonMgr::GetInstance().GetExtValue(ext->configuration, "@extension");
   std::string protocols = CAddonMgr::GetInstance().GetExtValue(ext->configuration, "@protocols");
   std::string name(ext->plugin->identifier);
+
+  bool matchAPIVersion(true);
+  for (unsigned int i(0); i < ext->plugin->num_imports; ++i)
+    if (strcmp(ext->plugin->imports[i].plugin_id,"kodi.inputstream")==0 && strcmp(ext->plugin->imports[i].version, INPUTSTREAM_API_VERSION))
```

Comment:
Please add != 0 to the second statement, as people not knowing what strcmp returns lead to confusions.